### PR TITLE
Important websites operating in Poland.

### PR DESCRIPTION
**Pull request by dMbski**
I noticed that users liked the favicon, so I added the websites that operate in Poland.

### CODE REVIEW

File: data/favicon/intercity.pl.png
Comment (lieser) [inline data/favicon/intercity.pl.png:1]: Consider using the 32x32 version instead (https://www.intercity.pl/favicon-32x32.png), which I use most of the time as a compromise between size and visible details.

But also OK with leaving as is.

File: data/favicon/onet.pl.ico
Comment (lieser) [inline data/favicon/onet.pl.ico:1]: I currently see a different icon when visiting their website (just a yellow circle). Did they recently change their logo?

**Comment by dMbski** [comment]
Favicon is from e-mail service, which has this logo poczta.onet.pl. 
  Dnia 17 września 2023 15:57 Philippe Lieser   napisał(a):  
                 
 @lieser  commented on this pull request. 

 

 On  github.com data/favicon/intercity.pl.png : 
 Consider using the 32x32 version instead ( www.intercity.pl https://www.intercity.pl/favicon-32x32.png ), which I use most of the time as a compromise between size and visible details. 
 But also OK with leaving as is. 

 

 On  github.com data/favicon/onet.pl.ico : 
 I currently see a different icon when visiting their website (just a yellow circle). Did they recently change their logo?

**Comment by lieser** [comment]
> Favicon is from e-mail service, which has this logo poczta.onet.pl.

Am I understanding it correctly that this is an e-mail provider, and users are getting e-mail addresses from the `onet.pl` domain? Than note that a general rule for domains of e-mail providers is not allowed. Only for specific official and important e-mail addresses such an icon could be added (or an AUID in case they are using that).

More details on the requirements for favicon are described here:
https://github.com/lieser/dkim_verifier/wiki/Contribute#additional-requirements-for-favicon

File: data/favicon/onet.pl.ico
Comment (dMbski) [reply data/favicon/onet.pl.ico:1]: They change the logo for various events and. The logo I provided is from the website poczta.onet.pl, to which users log in to use e-mail. Maybe the icon from the address https://ocdn.eu/pocztastatic/pocztareact/favicons/favico_default.png, which is displayed after logging in to the onet.pl email service, will be more appropriate. 
Załączyłem plik ekranem jak to wygląda w przeglądarce.
poczta-onet-pl-after-login

File: data/favicon/intercity.pl.png
Comment (dMbski) [reply data/favicon/intercity.pl.png:1]: Yes, this 32pixels icon is better.

**Comment by dMbski** [comment]
In my rush and distraction, I didn't read your note about not **adding icons to email provider services**. 
So I **deleted** them. Sorry about this.

The new icons concern important utility services.

**Review by lieser — APPROVED** [review]
Thanks for adding the icons, LGTM now.

**Pull request merged**